10 - Hold Shift and Check Checkboxes/index-START.html

```diff
@@ -96,6 +96,33 @@
   </div>
 
 <script>
+  //// global variables
+  let selectedInput;
+
+  //// selectors
+  const checkboxes = document.querySelectorAll('.item input');
+  
+  //// callbacks
+  function selectItems(evt) {
+    let isAmongTheSelected = false;
+
+    if (evt.shiftKey && this.checked) {
+      checkboxes.forEach(checkbox => {
+        if (checkbox === this || checkbox === selectedInput) {
+          isAmongTheSelected = !isAmongTheSelected;
```

Comment:
is it ok that `isAmongTheSelected` is not changed from the last for-iteration if the condition is not fulfilled?
if not, I am not sure if we should use a variable alltogether.

Reply:
`isAmongTheSelected` is just a `boolean` that flags the inputs if they are between the first and last `selectedInput`. I did not find any smarter way to make it work and without it does not work for me. 

The `forEach` is basically looking for the first available `isAmongTheSelected === false` and stop assigning `checked = true` because the last selected input has been already `checked = true` by the user with `shiftKey`.

I'm struggling to find another way to implementing it without the variable.

Any tips?